sources/packages/backend/apps/api/src/route-controllers/application/application.controller.service.ts

```diff
@@ -107,8 +107,8 @@ export class ApplicationControllerService {
         data.selectedProgram,
       );
       if (selectedProgram) {
-        // Assign selected program && selected offering for application as null when the program is inactive.
-        if (!selectedProgram.isActive) {
+        // Assign selected program && selected offering for application as null when the program is inactive or expired.
+        if (!selectedProgram.isActive || selectedProgram.isExpired) {
```

Comment:
I am not sure why it was previously separate in two `if` conditions but can we have the changed as below?
```ts
if (!selectedProgram.isActive
  || selectedProgram.isExpired
  || selectedProgram.programStatus !== ProgramStatus.Approved) {
   data.selectedProgram = null;
}
```